src/App.tsx

```diff
@@ -1,61 +1,122 @@
+import { ChangeEvent, MouseEvent, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+const initialTodos = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [todos, setTodos] = useState(initialTodos);
+  const [title, setTitle] = useState('');
+  const [isValidTitle, setIsValidTitle] = useState(false);
+  const [userId, setUserId] = useState(0);
+  const [isValidUser, setIsValidUser] = useState(false);
+
+  let count = 15;
```

Comment:
1. count of what?
2. using let inside compoent is bad practice - > move it to a state or use const and move it out of component

Reply:
I am not sure you really need it at all